{u'description': [u'Personal User Overlay'],
 u'feeds': [u'https://github.com/aliceinwire/overlay/commits/master.atom'],
 u'homepage': u'https://github.com/aliceinwire/overlay',
 u'irc': None,
 u'name': u'pypy_alice',
 u'official': False,
 u'owner_email': u'alicef@gentoo.org',
 u'owner_name': u'Alice Ferrazzi',
 u'priority': 50,
 u'quality': u'experimental',
 u'sources': [(u'git://github.com/aliceinwire/overlay.git', u'Git', u''),
              (u'https://github.com/aliceinwire/overlay.git', u'Git', u''),
              (u'git@github.com:aliceinwire/overlay.git', u'Git', u'')],
 u'status': u'unofficial',
 u'supported': True}
$ pmaint sync pypy_alice
WARNING:pkgcore:repository at '/home/mgorny/repos/steveeJ' has unsupported profile format(s): 'profile-set'
Already up-to-date.
*** syncing pypy_alice
*** synced pypy_alice
 * Sync succeeded
$ pmaint regen -t 2 pypy_alice
WARNING:pkgcore:repository at '/home/mgorny/repos/steveeJ' has unsupported profile format(s): 'profile-set'
WARNING:pkgcore:repository at '/home/mgorny/repos/steveeJ' has unsupported profile format(s): 'profile-set'
Error running an operation:
 Unhandled exception in regen_cache
 [Errno 13] Permission denied
coms error in 30563, read_line com failed w/ 1: backing out of daemon.
 * Cache regen failed with 246
